src/layers/layer-update.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable guard-for-in */
+
+/**
+ *
+ * @param {Object} updateTriggers {getPosition: {column}, getData: {filteredIndex}}
+ * @param {Object} oldUpdateTriggers
+ * @returns {}
+ */
+export function diffUpdateTriggers(updateTriggers, oldUpdateTriggers = {}) {
+  const triggerChanged = {};
+  let reason = false;
```

Comment:
You shouldn't use false to set a variable which later one you will use to store an object
```
triggerChanged[triggerName] = true;
reason = triggerChanged;
```

I think you can also change the name to reasons since you may have multiple properties in the object